uint32 value
---
int8 success
